docs/development_environment.md

```diff
@@ -46,130 +44,52 @@ It is also possible to set up a more traditional development environment. See th
-
-```shell
-$ mkdir -p ../config
-$ hass -c ../config
-```
+To develop on Windows, you will need to use the Linux subsystem. Follow the [WSL installation instructions](https://docs.microsoft.com/windows/wsl/install-win10) and install Ubuntu from the Windows Store. Once you're able to access Linux, follow the Linux instructions.
 
-:::tip
-You may find that you cannot open the development instance via <http://localhost:8123> when using WSL. Instead, within a WSL terminal, find the `inet` address of the `eth0` adaptor by running `ip addr show eth0`. Then use this address, excluding the CIDR block, to access the development instance, i.e. if your `inet` is listed as `172.20.37.6/20`, use <http://172.20.37.6:8123>.
-:::
+If you find that you cannot open the development instance via <http://localhost:8123> when using WSL. Instead, within a WSL terminal, find the `inet` address of the `eth0` adaptor by running `ip addr show eth0`. Then use this address, excluding the CIDR block, to access the development instance, i.e. if your `inet` is listed as `172.20.37.6/20`, use <http://172.20.37.6:8123>.
 
 ### Developing on macOS
 
-Install [Homebrew](https://brew.sh/), then use that to install Python 3:
-
-```shell
-$ brew install python3 autoconf
-```
-
-Then install ffmpeg:
+Install [Homebrew](https://brew.sh/), then use that to install the dependencies:
 
 ```shell
-$ brew install ffmpeg
+$ brew install python3 autoconf ffmpeg
 ```
 
 ## Setup Local Repository
 
 Visit the [Home Assistant Core repository](https://github.com/home-assistant/core) and click **Fork**.
 Once forked, setup your local copy of the source using the commands:
 
-_Windows users should be sure to clone to a path that inside the WSL (ex: ~/)._
-
 ```shell
 $ git clone https://github.com/YOUR_GIT_USERNAME/core.git
 $ cd core
 $ git remote add upstream https://github.com/home-assistant/core.git
 ```
 
-Note that `core.git` should be replaced by the name of your fork (default being `core.git`). If unsure check your GitHub repository.
-
-## Setting Up Virtual Environment
-
-To isolate your environment from the rest of the system, set up a [`venv`](https://docs.python.org/3/library/venv.html). Within the `core` directory, create and activate your virtual environment.
+Install the requirements with a provided script named `setup`.
 
 ```shell
-$ python3.7 -m venv venv
-$ source venv/bin/activate
+script/setup
 ```
 
-Install the requirements with a provided script named `setup`.
+This will create a virtual environment and install all necessary requirements. You're now set! 
+
+Each time you start a new terminal session, you will need to activate your virutal environment:
 
 ```shell
-$ script/setup
+source venv/bin/activate
 ```
 
-Invoke your installation, adjusting the [configuration](https://www.home-assistant.io/docs/configuration/) if required.
+After that you can run Home Assistant like this:
 
 ```shell
-$ hass
+hass -c config
```

Comment:
Maybe, we should make the config loading/search of HA look for a local `config` folder first, next the other options currently already there.

If none exists and config is created, skip the local `config` folder as a possible option to create the config files.

Reply:
I think that would be too much of a breaking change right now.